libpromises/evalfunction.c

```diff
@@ -6888,6 +6889,47 @@ static int BuildLineArray(EvalContext *ctx, const Bundle *bundle,
 
 /*********************************************************************/
 
+static FnCallResult FnCallProcessExists(ARG_UNUSED EvalContext *ctx, ARG_UNUSED const Policy *policy, const FnCall *fp, const Rlist *finalargs)
+{
+    char *regex = RlistScalarValue(finalargs);
+
+    const bool context_mode = strcmp(fp->name, "processexists") == 0;
```

Comment:
Would be nice to have something more descriptive here like is_context_processexists or something similar.